### PR TITLE
A kick command

### PR DESCRIPTION
**Pull request by mgejay**
^^^

### CODE REVIEW

**Comment by Hockeyfan360** [comment]
Three commits in one pr, you need to test it out next time. As well as that, this is not in allman style, so this PR is invalidated. Please open a new pr when you are ready to merge again. USE NETBEANS AND TEST BEFORE SENDING IN A PULL REQUEST. (The main reason I am closing this is because there are already three commits to fix the same issue, open a new pull request combining all of your fixes into one, and only commit after testing.)

File: src/com/superiornetworks/icarus/commands/Command_kick.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandParameters(name="kick",description="Kicks a player",usage="/kick <player> <reason>",rank=ICM_Rank.Rank.SUPER)
+public class Command_kick
+{   
+    
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_kick.java:18]: Not allman style.

File: src/com/superiornetworks/icarus/commands/Command_kick.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        if (args.length < 2) {
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_kick.java:19]: Not allman style.

**Comment by mgejay** [comment]
I tested @Hockeyfan360, this worked.

**Comment by Hockeyfan360** [comment]
It would be a fourth commit for one PR, it would be best to open a new Pull Request to do this.

**Pull request closed without merging**